### ISSUE TITLE
Sharing

### ISSUE THREAD

**Issue by chrisegan22** [nextcloud/android#7932]
### Steps to reproduce
1. Share in app click link
2. 
3. 

### Expected behaviour
- Tell us what should happen

### Actual behaviour
- Tell us what happens

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by thgoebel** [comment]
Could be related to #7987. Do you see a crash log after selecting the user to share the file with?

**Comment by chrisegan22** [comment]
No there is no error log but it doesn't crash any more but you cant send folders only loose files if you now try to send a folder it doesn't do anything. The link feature copies the link

**Comment by tobiasKaminsky** [comment]
Duplicate of https://github.com/nextcloud/android/issues/7987
Fix is in: #7992
